src/components/Home/Hero/ShowcaseCarousel.tsx

```diff
@@ -1,9 +1,13 @@
-import "@/styles/hero-carousel.module.css";
 import { ArrowDown } from "@phosphor-icons/react";
 import "keen-slider/keen-slider.min.css";
 import { useKeenSlider } from "keen-slider/react";
+import { mockDataImage } from "@/mock/mockdataImage";
+import Image from "next/image";
 
 export default function ShowcaseCarousel() {
+
+  const heroSlider = "flex items-center justify-center text-[50px] text-white font-medium min-w-[459.35px] min-h-[424.48px] max-h-[459.35px]"
```

Comment:
I think this should be placed directly on the HTML element it should contain. Having one long line with tailwind is inevitable I guess if it really have many styles in it.